docs/subvolume.md

```diff
@@ -0,0 +1,45 @@
+# Subvolume cleanup
+
+The subvolume command is used to clean the stale subvolumes
+which has no parent-pvc attached to it.
+The command would list out all such subvolumes which needs to be removed.
+This would consider all the cases where we can have stale subvolume
+and delete them without impacting other resources and attached volumes.
+
+The subvolume command will require the following sub commands:
+* `ls` : [ls](#ls) lists all the subvolumes
+    * `--type=stale`: lists all the stale volume that can be deleted
+* `delete` : [delete](#delete) stale subvolumes after user's confirmation.
+    * It takes 3 arguments:
+      * list of subvolumes of same filesystem and subvolumegroup
+      * filesystem name
+      * subvolumegroup name
+
+## ls
+
+```bash
+kubectl rook-ceph subvolume ls
+
+# csi-vol-427774b4-340b-11ed-8d66-0242ac110004
```

Comment:
What about printing two columns? `Subvolume\tStatus`. For status we could have something like `stale` or `bound`
```suggestion
# SUBVOLUME                                     STATUS
# csi-vol-427774b4-340b-11ed-8d66-0242ac110004  stale
```

Reply:
not sure how it will be done, need to check.

Reply:
Anyways, I dont think it is required, cause we already have a command that lists the stale subvolumes , so doing it in ls also won't make much sense to me.